src/app/screens/Settings.tsx

```diff
@@ -29,13 +29,47 @@ function Settings() {
   const [modalIsOpen, setModalIsOpen] = useState(false);
   const [formData, setFormData] = useState(initialFormData);
 
+  // TODO: Load setting from encrypted storage
+  const [nostrPrivateKey, setNostrPrivateKey] = useState("");
+
+  const getPrivateKeyFromStorage = async () => {
+    const priv = (await utils.call("nostr/getPrivateKey")) as string;
+    setNostrPrivateKey(priv ?? "");
+  };
+
+  useEffect(() => {
+    getPrivateKeyFromStorage().catch(console.error);
```

Comment:
Will this automatically console an error?

Reply:
yes, if for instance the private key could not be decoded with the password, etc